remoulade/worker.py

```diff
@@ -37,6 +37,14 @@
 CONSUMER_RESTART_MAX_RETRIES = int(os.getenv("remoulade_restart_max_retries", 10))
 
 
+def build_extra(message):
```

Comment:
maybe add a `max_input_size=None` parameter, so it's easily customizable, and use for both "classic" logs with a limit of 1000 and for the "exception" log without limit